missions/W5/M2/nyc_taxi_analysis.py

```
@@ -0,0 +1,78 @@
+import pyspark
+from pyspark.sql import SparkSession
+from pyspark.sql import Row
```

Ruff (0.8.2)

`pyspark.sql.Row` imported but unused

Remove unused import: `pyspark.sql.Row`

(F401)